src/components/ReadMeDoc.js

```diff
@@ -0,0 +1,18 @@
+import React, { useEffect, useState } from 'react'
+import ReactMarkdown from 'react-markdown'
+
+export default function ReadMeDoc({ docURL }) {
+	const [markdown, setMarkdown] = useState('')
+
+	useEffect(() => {
+		fetch(docURL)
+			.then((res) => res.text())
+			.then((text) => setMarkdown(text))
+	}, [])
+
+	return (
+		<>
+			<ReactMarkdown children={markdown} />
```

Comment:
What happens if url is unavailable or broken?

Reply:
Good point!
Just implemented a catch with an Error message, asking to go to github doc instead.